PWGLF/Tasks/Nuspex/NucleiHistTask.cxx

```diff
@@ -507,8 +543,8 @@ struct NucleiHistTask {
         proton_reg.fill(HIST("histTpcNsigmaData"), momentum, nSigmaProton);
         deuteron_reg.fill(HIST("histTpcNsigmaData"), momentum, nSigmaDeut);
         triton_reg.fill(HIST("histTpcNsigmaData"), momentum, nSigmaTriton);
-        Helium3_reg.fill(HIST("histTpcNsigmaData"), momentum * 2.0, nSigmaHe3);
-        Helium4_reg.fill(HIST("histTpcNsigmaData"), momentum * 2.0, nSigmaHe4);
+        Helium3_reg.fill(HIST("histTpcNsigmaData"), momentum_Z_equal_2 * 2.0, nSigmaHe3);
```

Comment:
Just a small doubt before merging: is this multiplication by 2 still needed now that you are already multiplying in the switch-case above?

Reply:
Thanks for noticing, I overlooked it.